src/Utilities/Compiler/HashUtilities.cs

```diff
@@ -9,7 +9,7 @@ namespace Analyzer.Utilities
 {
     internal static class HashUtilities
     {
-        internal static int GetHashCodeOrDefault(this object? objectOpt) => objectOpt?.GetHashCode() ?? 0;
+        internal static int GetHashCodeOrDefault(this object? @object) => @object?.GetHashCode() ?? 0;
```

Comment:
For similarity with `EqualityComparer<T>.GetHashCode(T)`

```suggestion
        internal static int GetHashCodeOrDefault(this object? obj) => obj?.GetHashCode() ?? 0;
```